frontend/src/components/EventForm/EventForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Box, Typography } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import createNewEvent from '../../requests/post/createNewEvent';
+import Alert from '@mui/material/Alert';
+import './style.css';
+
+function EventForm() {
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [date, setDate] = useState("");
+    const [time, setTime] = useState("");
+    const [image, setImage] = useState("");
+
+    const [successAlert, setSuccessAlert] = useState(false);
+    const [errorAlert, setErrorAlert] = useState(null);
+
+    const handleSubmit = () => {
```

Comment:
Input fields and submit button should be disabled while the request is being processed

Reply:
All input fields are no disabled until we receive the response.